Ruby/Ruby.sublime-syntax

```diff
@@ -1252,17 +1252,14 @@ contexts:
       scope: constant.character.escape.ruby
 
   interpolated-ruby:
-    - match: '(#\{)(\})'
-      scope: punctuation.section.embedded.ruby
-      captures:
-        1: source.ruby.embedded.source
-        2: source.ruby.embedded.source.empty
     - match: '#\{'
-      scope: punctuation.section.embedded.ruby
+      scope: punctuation.definition.interpolation.begin.ruby
```

Comment:
Via https://github.com/sublimehq/Packages/pull/1432#issuecomment-366750733, I believe `punctuation.section.interpolation.*` should be used.

If this is not currently the case with Python, it should be adjusted imo.

Reply:
@Thom1729 this should be

```suggestion
      scope: punctuation.section.interpolation.begin.ruby
```